app/constants/event-details.js

```diff
@@ -0,0 +1,128 @@
+const {
+  PAYMENT_EXTRACTED,
+  PAYMENT_ENRICHED,
+  PAYMENT_PAUSED_DEBT,
+  PAYMENT_DEBT_ATTACHED,
+  PAYMENT_PAUSED_LEDGER,
+  PAYMENT_LEDGER_ASSIGNED,
+  PAYMENT_PAUSED_QUALITY_CHECK,
+  PAYMENT_QUALITY_CHECK_FAILED,
+  PAYMENT_QUALITY_CHECK_PASSED,
+  PAYMENT_RESET,
+  PAYMENT_PROCESSED,
+  PAYMENT_SUBMITTED,
+  PAYMENT_ACKNOWLEDGED,
+  PAYMENT_SETTLED
+} = require('./events')
+const {
```

Comment:
Just noticed a space is needing before this line

Reply:
Was sure I did that! going code blind after this week...

Done